.github/workflows/update_dependencies.yml

```diff
@@ -101,7 +101,7 @@ jobs:
           body: "Firebase Api update"
           labels: |
             automated pr
-            firease_api
+            firebase_api
           reviewers: adamvduke,jakeouellette,tonybaroneee
```

Comment:
please replace `adamvduke` with `kaibolay`

Reply:
There are 3 instances, do you want all 3 changed, or just this one?

https://github.com/Flank/flank/blob/7a9163b2aba43102da960ffbedf612dc114a93af/.github/workflows/generate_release_notes.yml#L58
https://github.com/Flank/flank/blob/7a9163b2aba43102da960ffbedf612dc114a93af/.github/workflows/update_dependencies.yml#L60
https://github.com/Flank/flank/blob/7a9163b2aba43102da960ffbedf612dc114a93af/.github/workflows/update_dependencies.yml#L105

Reply:
All of them, please.